.github/workflows/main.yml

```diff
@@ -331,6 +337,25 @@ jobs:
         name: Renders_${{ matrix.name }}
         path: build/render/*.png
 
+    - name: Archive Resources (macOS)
+      if: matrix.test_render == 'ON' && runner.os == 'macOS'
+      run: |
+        mkdir -p build/artifacts
+        if [ ! -d build/bin/resources ]; then
+          echo "Missing build/bin/resources"
+          echo "Contents of build/bin (if present):"
+          ls -la build/bin || true
+          exit 1
+        fi
+        zip -r build/artifacts/resources.zip build/bin/resources
```

Comment:
What is the motivation for _manually_ generating a zip archive of the folder, rather than leveraging the built-in zip functionality in `actions/upload-artifact`?

Would it make sense to align this with our other upload steps (e.g. Upload Reference Shaders, Upload Renders) and use the existing system in `actions/upload-artifact`?

Reply:
Good catch. I had this "cherry picking" images at one point but reverted that as it's better to produce a "results" folder as part of test. Will revert this to the simpler rule.